src/components/ListSettingsItem.js

```diff
@@ -66,7 +73,13 @@ export const ListSettingsItem = ({ item }) => {
             onPress={getOnPressListType(listType)}
             delayPressIn={0}
             Component={Touchable}
-            accessibilityLabel={`(${texts.settingsTitles.listLayouts[listType]}) ${consts.a11yLabel.button}`}
+            accessibilityLabel={`(${texts.settingsTitles.listLayouts[listType]}) ${
+              consts.a11yLabel.button
+            } (${
+              listType === listTypeForQuery
+                ? texts.accessibilityLabels.tabs.active
+                : texts.accessibilityLabels.tabs.inactive
+            })`}
```

Comment:
can this format be optimized? why is the `consts.a11yLabel.button` multilined? does our linter/formatter do it automatically?

Reply:
Unfortunately, this is how it looks after automatic formatting. I will try to write in a more readable format.

Reply:
ok, if it is automatically changed this way, we cannot handle it

Reply:
I guess it's easier to read now

https://github.com/smart-village-solutions/smart-village-app-app/commit/f235b9769c1abcef45eb059543897a509bea090f